index.html

```diff
@@ -51,9 +51,220 @@ <h2 class="hero-msg">Glad to see you!</h2>
         </div>
       </div>
       <div>
-        <a href="#" class="text-connect">
-          <h5>LET'S CONNECT</h5>
-        </a>
+        <a href="#" class="text-connect"> LET'S CONNECT </a>
+        <div>
+          <ul class="icons-hero">
+            <li>
+              <img src="./img/twitter.svg" alt="twitter logo" />
+            </li>
+            <li>
+              <img src="./img/linkedin.svg" alt="linkedin logo" />
+            </li>
+            <li>
+              <img src="./img/medium.svg" alt="medium logo" />
+            </li>
+            <li>
+              <img src="./img/github.svg" alt="github logo" />
+            </li>
+            <li>
+              <img src="./img/angellist.svg" alt="angellist logo" />
+            </li>
+          </ul>
+        </div>
+      </div>
+    </main>
+    <section>
+      <div class="card-container">
+        <div class="card-wrapper">
+          <div class="card">
+            <div class="card-image">
+              <img src="./img/Portfolio.svg" alt="portfolio" />
+            </div>
+            <div class="card-text-container">
+              <div class="card-text">
+                <h6>TONIC</h6>
+              </div>
+              <div class="card-tagline">
+                <div class="card-canopy">
+                  <h6>CANOPY</h6>
+                </div>
+                <div class="card-dot">
+                  <img src="./img/dot.png" alt="dot" />
+                </div>
+                <div class="card-job">
+                  <h6>Back End Dev</h6>
+                </div>
+                <div class="card-dot">
+                  <img src="./img/dot.png" alt="dot" />
+                </div>
+                <div class="card-job">
+                  <h6>2015</h6>
+                </div>
+              </div>
+              <div class="card-desc">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </div>
```

Comment:
- [ ] Kindly replace the `div` with a `p` tag.
- The `p` tag is used to define a paragraph of text. When you use the `p` tag, it tells the browser that this content is a paragraph and should be displayed as such. Using <p> tags also help search engines understand the structure of your page and improve accessibility for screen readers.

On the other hand, the `div` tag is a generic container element that is used to group other elements together. It has no semantic meaning on its own. You can use `div` tags to group related elements together and apply styles or scripts to them as a group.

Reply:
done :heavy_check_mark: